lazy_tensor_core/lazy_tensor_core/csrc/ops/scalar.h

```diff
@@ -30,6 +30,7 @@ class Scalar : public TsNode {
 
 torch::lazy::hash_t ScalarHash(const at::Scalar& s);
 
+// TODO: remove this after the aten version is exposed properly
```

Comment:
I was having a hard time to access the same function from `#include <ATen/core/Formatting.h>`. We might need to change pytorch core to access it.

Reply:
huh.  namespace issue?

Reply:
Why remove it?

Reply:
> Why remove it?

I think it is a temp workaround from Alex. We could just use the one from https://github.com/pytorch/pytorch/blob/f5daa9f76bcf31bab61feefc668de523bc667c2f/aten/src/ATen/core/Formatting.h#L23

Reply:
> huh. namespace issue?

Yes. I have uploaded a new version which works now after adding `using namespace at;`. I initially put that in scalar.cpp so it did not work...